tidb-cloud/explore-data-with-chat2query.md

```diff
@@ -9,12 +9,11 @@ TiDB Cloud is powered by AI. You can use Chat2Query (beta), an AI-powered SQL ed
 
 In Chat2Query, you can either simply type `--` followed by your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal. You can find the query results in tables intuitively and check the query logs easily.
 
-> **Note:**
->
-> Chat2Query is supported for TiDB clusters that are v6.5.0 or later and are hosted on AWS.
->
-> - For [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless) clusters, Chat2Query is available by default.
-> - For [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters, Chat2Query is only available upon request. To use Chat2Query on TiDB Dedicated clusters, contact [TiDB Cloud support](/tidb-cloud/tidb-cloud-support.md).
+## Limitations
+
+- SQL queries generated by the AI may not be 100% accurate, and you may need to refine them.
```

Comment:
```suggestion
- SQL queries generated by the AI might not be 100% accurate, and you might need to refine them.
```

Reply:
In general, "may" is reserved for official policy or legal considerations. To convey possibility, use can or might instead.

ref: https://developers.google.cn/style/word-list#may